templates/powershell/build/ImportSecretsToKV.ps1

```diff
@@ -20,6 +20,19 @@ param(
     [string]$PSHelperDirectory
 )
 
+function GetDecodedValue {
```

Comment:
do we need to decode and encode? i think you can use $secret var directly ?

Reply:
Removed it